src/bmffimage.cpp

```diff
@@ -56,7 +56,8 @@
 #define TAG_heix 0x68656978 /**< "heix" HEIC */
 #define TAG_mif1 0x6d696631 /**< "mif1" HEIF */
 #define TAG_crx  0x63727820 /**< "crx " Canon CR3 */
-#define TAG_jxl  0x4a584c20 /**< "JXL " JPEG XL   */
+#define TAG_JXL  0x4a584c20 /**< "JXL " JPEG XL */
```

Comment:
Looking at #1503, the `ftyp` should be `jxl` so good catch, and thanks for the fix!  
But I don't think we need the additional `TAG_JXL`

Reply:
Indeed, the "JXL " magic signature in capitals is handled separately [here](https://github.com/Exiv2/exiv2/blob/686702616c33267217234b2189d48a42b8ad723d/src/bmffimage.cpp#L649-L650).

Reply:
@kmilos Hvala.